src/robotremoteserver.py

```diff
@@ -70,7 +70,8 @@ def __init__(self, library, host='127.0.0.1', port=8270, port_file=None,
                             ``Stop Remote Server`` keyword and
                             ``stop_remote_server`` XML-RPC method.
         """
-        self._library = RemoteLibraryFactory(library)
+        self._library = [RemoteLibraryFactory(library_) 
```

Comment:
```suggestion
        if isinstance(libraries, list):
            self._libraries = [RemoteLibraryFactory(library_) 
                         for library_ in libraries]
        else:
            self._libraries = [RemoteLibraryFactory(libraries)]
```

Though not sure if checking `isinstance` is the most pythonic, maybe checking to see if the item supports ` __iter__` would be more robust?

Reply:
I think this solutions works. Best if you create a new PR with these changes.